libr/core/cmd_debug.c

```diff
@@ -411,6 +411,10 @@ static void cmd_debug_pid(RCore *core, const char *input) {
 		"dpt=", "<thread>", "Attach to thread",
 		NULL};
 	switch (input[1]) {
+	case 0:
+		eprintf ("Selected: %d %d\n", core->dbg->pid, core->dbg->tid);
```

Comment:
Probably better to use rconsprintf here, otherwise this information will not be catched in r2pipe